build-with-pip.file

```diff
@@ -0,0 +1,25 @@
+%define my_name %(echo %n | cut -f2 -d-)
+Source: none
+
+Requires: python
+BuildRequires: py2-pip
+
+%prep
+
+%build
+
+pip list
+pip install --no-deps -t %{i}/$PYTHON_LIB_SITE_PACKAGES  %{my_name}==%{realversion}
```

Comment:
@davidlange6 , try the following. It allows pip to use our py2 tools
```
export PYTHONUSERBASE=%i
pip list
pip install --user %{my_name}==%{realversion}
```
and pip sysy
```
+ pip install --user entrypoints==0.2.2
Collecting entrypoints==0.2.2
  Using cached entrypoints-0.2.2-py2.py3-none-any.whl
Requirement already satisfied: configparser>=3.5; python_version == "2.7" in /var/build/muz/pip/w/slc6_amd64_gcc530/external/py2-configparser/3.5.0/lib/python2.7/site-packages (from entrypoints==0.2.2)
Installing collected packages: entrypoints
Successfully installed entrypoints-0.2.2
```